website/docs/getting-started/setup/instance-configuration/authentication/github-login.md

```diff
@@ -27,28 +27,43 @@ To enable GitHub Sign in, login to your [GitHub Account](https://github.com)
 
 **4. Configure the OAuth Credentials for a web application**
 
-**Homepage URL:** The domain on which you are hosting Appsmith with https **`(ex: https://app.appsmith.com)`**
+**Homepage URL:** the domain on which you are hosting Appsmith with HTTPS **`(ex: https://app.appsmith.com)`**
 
-**Authorization Callback URL:** Append **/login/oauth2/code/github** to the Homepage URL **`(ex: https://app.appsmith.com/login/oauth2/code/github)`**
+**Authorization Callback URL:** append **/login/oauth2/code/github** to the Homepage URL **`(ex: https://app.appsmith.com/login/oauth2/code/github)`**
 
 ![Click to expand](</img/Github_App_Config.png>)
 
 **5. Update the** [**instance configuration**](../) **with the below fields**
 
-```bash
-# Example Docker Configuration 
-# ********* Github OAUth **********
-APPSMITH_OAUTH2_GITHUB_CLIENT_ID=YOUR_GITHUB_CLIENT_ID
-APPSMITH_OAUTH2_GITHUB_CLIENT_SECRET=YOUR_GITHUB_CLIENT_SECRET                                                                                                                          18,61         28%
-# ******************************
-```
+  * **Configure via UI (recommended)**:
 
-:::caution
-The Client ID and Client Secret entered are not verified by any means. Please test it out manually before disabling the Form login authentication method. If Form login is disabled before testing the GitHub authentication config setup, it may lead to the user getting stuck on the login screen if the GitHub config is invalid. To fix this, you’ll have to update the ENV variables manually by adding a valid Client ID and Client Secret or enabling the form login. You won’t be able to do this via the Admin settings page since you won’t be logged in anymore.
-:::
+  On your self-hosted instance, navigate to the [Admin Settings page](/getting-started/setup/instance-configuration/admin-settings#using-the-admin-settings-ui). Under the **Authentication** tab, you can find fields for providing your Client ID and Client Secret generated in the previous steps.
+
+  ![](/img/as_github_auth_config.png)
+
+  Once you've entered your provided the Client ID and Client Secret, click the **Save & Restart** button at the bottom of the page. This prompts your Appsmith instance with the new configuration in effect.
+
+  * **Configure via docker.env file**:
+
+  On the machine hosting the Appsmith instance, find your `docker.env` file located in `<project-root>/stacks/configuration/`. Update the values for the following keys:
 
-**6.** [**Restart the Appsmith instance**](../)
+    ```bash
+    # Example configuration in docker.env file
+    # ********* Github OAUth **********
+    APPSMITH_OAUTH2_GITHUB_CLIENT_ID=YOUR_GITHUB_CLIENT_ID
+    APPSMITH_OAUTH2_GITHUB_CLIENT_SECRET=YOUR_GITHUB_CLIENT_SECRET
+    # ******************************
+    ```
 
-:::tip
```

Comment:
why are we removing this tip?

Reply:
Oops; I had a reason for turning this tip into plain text (it appears at the bottom of doc) but I don't remember exactly why I did it. Looking at it again now, it should probably stay as a tip and move up a bit, I'll adjust that